epi/running.py

```diff
@@ -0,0 +1,18 @@
+"""Run parsing on a list of inputs."""
+from epi.epi_functions import dump_epi_tb4
+
+dump_epi_tb4(datum="2020-10-15", data_loc="./data")
+dump_epi_tb4(datum="2020-10-16", original_csv="epi_tb4_dump_2020-10-15.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-17", original_csv="epi_tb4_dump_2020-10-16.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-18", original_csv="epi_tb4_dump_2020-10-17.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-19", original_csv="epi_tb4_dump_2020-10-18.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-20", original_csv="epi_tb4_dump_2020-10-19.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-21", original_csv="epi_tb4_dump_2020-10-20.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-22", original_csv="epi_tb4_dump_2020-10-21.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-23", original_csv="epi_tb4_dump_2020-10-22.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-24", original_csv="epi_tb4_dump_2020-10-23.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-25", original_csv="epi_tb4_dump_2020-10-24.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-26", original_csv="epi_tb4_dump_2020-10-25.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-27", original_csv="epi_tb4_dump_2020-10-26.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-28", original_csv="epi_tb4_dump_2020-10-27.csv", data_loc="./data")
+dump_epi_tb4(datum="2020-10-29", original_csv="epi_tb4_dump_2020-10-28.csv", data_loc="./data")
```

Comment:
What is the purpose of this file? For testing or something to be run on a daily basis?
There are almost enough lines to warrant a `for` loop ;)

Reply:
Or even better we have a dedicated directory and fetch every file from there maybe?

Reply:
This is not meant for production. It was used for testing but I included it as a showcase.

@jurijbajzelj tell me more about how this would go.

Reply:
If it's not related to production you can disregard my comment :)